app/views/users/_users_form.html.erb

```diff
@@ -0,0 +1,65 @@
+<% if user.errors.any? %>
+  <div id="error_explanation">
+    <h2>
+      <%= pluralize(user.errors.count, "error") %>
+      prohibited this user from being saved:
+    </h2>
+
+    <ul>
+    <% user.errors.full_messages.each do |msg| %>
+      <li><%= msg %></li>
+    <% end %>
+    </ul>
+  </div>
+<% end %>
+
+<!--
+<form action="/users" method="post">
+
+  <label for="username">User name:</label>
+  <input type="text" name="user[username]">
+  <br>
+
+  <label for="email">email:</label>
+  <input type="text" name="user[email]">
+  <br>
+
+  <label for="password">Password</label>
+  <input type="password" name="user[password]">
+
+  <br>
+  <button type="submit">Submit</button>
+
+  <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>">
+
+</form>
+-->
+
+<%#= form_tag('/users', method: :post) do %>
+<%= form_for user do |usr| %>
```

Comment:
Better to use `@user` to make reference to the instance variable in your controller

[`OPTIONAL`]